lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2613,10 +2615,16 @@ static function ( $pseudo_selector ) use ( $selector ) {
 		$block_rules = '';
 
 		/*
-		 * 1. Separate the declarations that use the general selector
+		 * 1. Bespoke declaration modifiers:
+		 * - 'filter': Separate the declarations that use the general selector
 		 * from the ones using the duotone selector.
+		 * - 'background|background-size': set the html minHeight to 100%
```

Comment:
Tiny nit: `background-image` instead of `background-size` and refer to min-height by its kebab-case name since we're in PHP?

```suggestion
		 * - 'background|background-image': set the html min-height to 100%
```